lib/improver/tests/ensemble_calibration/ensemble_calibration/test_EstimateCoefficientsForEnsembleCalibration.py

```diff
@@ -587,23 +620,22 @@ def test_coefficient_values_for_gaussian_distribution(self):
         data = [4.55819380e-06, -8.02401974e-09,
                 1.66667055e+00, 1.00000011e+00]
 
-        current_forecast = self.current_temperature_forecast_cube
-
-        historic_forecasts = self.historic_temperature_forecast_cube
+        historic_forecast = self.historic_temperature_forecast_cube
```

Comment:
Can't you just pass in self.historic_temperature_forecast_cube and self.temperature_truth_cube in rather than doing this?

Reply:
Yes, I've updated this here and for the rest of the unit tests within this file, where possible, so as to avoid an intermediate variable.